src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -413,3 +413,70 @@ def do_observation(self, event) -> None:
         self.responses = pd.concat(
             [self.responses, pop_included_mothers] + list(pop_included.values())
         )[self.output_columns]
+
+
+class SocialSecurityObserver(BaseObserver):
+    """Class for observing columns relevant to Social Security registry.
+
+    """
+
+    ADDITIONAL_INPUT_COLUMNS = ["alive", "entrance_time", "exit_time", "ssn"]
```

Comment:
You will need to include `"tracked"` as an input column, because otherwise an implicit `tracked == True` query is applied when you do `self.population_view.get()` and you need to be able to record the SSNs of people who become untracked during the sim.

Reply:
👍